x-pack/platform/plugins/shared/streams/server/lib/streams/state_management/execution_plan/types.ts

```diff
@@ -125,6 +126,14 @@ export interface DeleteDotStreamsDocumentAction {
   };
 }
 
+export interface SetFailureStoreAction {
+  type: 'set_failure_store';
```

Comment:
nit but can we name it `update_failure_store` to follow naming convention of the other actions